_episodes/01-introduction.md

```diff
@@ -117,11 +116,12 @@ When you are done with it, only then you may use the `plt.show()` as this will d
 There are different ways to create a legend in matplotlib but the easiest one to use would be to pass the keyword argument `label` inside the `plt.plot()` and use the `plt.legend()` command to automatically detect and show the individual labels on the canvas
 
 ```python
-plt.plot(x1, y1, label="log")
-plt.plot(x1, y2, label="cos")
-plt.plot(x1, y3, label="sin")
-
+fig, ax = plt.subplots()
+ax.plot(x1, y1, label="log")
+ax.plot(x1, y2, label="cos")
+ax.plot(x1, y3, label="sin")
 plt.legend()
+plt.show()
 ```
```

Comment:
I cannot comment in the unchanged part below, but in "The Figure", there's still some things to change. I think this should start with doing the `plt.subplots` with the arguments (instead of `plt.figure`) and then working from there

Reply:
Sorry I do not understand what needs to be done, I had tested this snippet with jupyter-notebook (it seems to work with and without plt.show statement. 

Could please let me know what I need to do here? thanks

Reply:
right I did see that earlier, But it requires many changes with the text as well.

Reply:
I will update the text and code then.

Reply:
As per PM: Will be addressed in 2nd PR